src/backend/rtlsym.d

```diff
@@ -0,0 +1,181 @@
+/**
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * Copyright:   Copyright (C) 1994-1998 by Symantec
+ *              Copyright (c) 2000-2016 by Digital Mars, All Rights Reserved
+ * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
+ * License:     backendlicense.txt
+ * Source:      $(DMDSRC backend/_rtlsym.d)
+ */
+
+module ddmd.backend.rtlsym;
+
+import ddmd.backend.cc : Symbol;
+
+enum
+{
+    RTLSYM_THROW,
```

Comment:
Are most of these enum values new? If so, please make them lowercase and replace the RTLSYM_ prefix with the name of the enum itself.